internal/mode/static/state/change_processor_test.go

```diff
@@ -1235,6 +1237,43 @@ var _ = Describe("ChangeProcessor", func() {
 				})
 			})
 		})
+		Describe("NginxProxy resource changes", Ordered, func() {
+			paramGC := gc.DeepCopy()
+			paramGC.Spec.ParametersRef = &v1beta1.ParametersReference{
+				Group:     ngfAPI.GroupName,
+				Kind:      v1beta1.Kind("NginxProxy"),
+				Name:      "np",
+				Namespace: helpers.GetPointer(v1beta1.Namespace("test")),
+			}
+
+			np := &ngfAPI.NginxProxy{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "np",
+					Namespace: "test",
+					Annotations: map[string]string{
```

Comment:
why do we need annotations?

Reply:
artifact from debugging tests. Will remove.